src/Materials/Textures/babylon.dynamicTexture.ts

```diff
@@ -35,6 +49,9 @@
             this._context = <CanvasRenderingContext2D>this._canvas.getContext("2d");
         }
 
+        /**
+         * Gets the current state of canRescale as true

```

Comment:
It will not always be true as child classes can overload it and returns false

Reply:
My lack of knowledge hits again. Will update.

Reply:
Absolutely no problem :)